kong/plugins/http-log/handler.lua

```diff
@@ -47,7 +47,7 @@ end
 function HttpLogHandler:log(conf)
   HttpLogHandler.super.log(self)
 
-  local route_id = conf.route_id or conf.api_id
+  local route_id = conf.route_id or conf.api_id or "global"
```

Comment:
Mind making this part of _another_ commit (could be the same PR, could be a different one), with its own test case (most importantly)? Thank you.

Reply:
That code change requires a global test case to produce a situation that could cause regression. Can I just make a new test case lua file where the plugin is defined as global? My most recent PR comment was around that problem.

Reply:
No need, you can write this test within a `describe` block that has a `setup` phase clearing up the database.

Reply:
@thibaultcha how we lookin now? I think I figured out how to make a new describes block and I believe to enable a plugin globally using this bp.plugins:insert{} you just drop setting the route id's, I struggled to find globally enabled plugin examples though during a 1 minute sweep searching the word global. Also broken out into 2 diff commits.

Reply:
It’s looking good @jeremyjpj0916! Just a few tweaks to adjust here and there :) I am confident this will make it in time for 1.0 :) Thank you!